kanidm_tools/src/cli/account.rs

```diff
@@ -23,7 +23,7 @@ impl AccountOpt {
                 AccountCredential::RemoveWebauthn(acs) => acs.copt.debug,
                 AccountCredential::RegisterTotp(acs) => acs.copt.debug,
                 AccountCredential::RemoveTotp(acs) => acs.copt.debug,
-                AccountCredential::GenerateBackupCode(acs) => acs.copt.debug,
+                AccountCredential::BackupCodeGenerate(acs) => acs.copt.debug,
```

Comment:
Sorry this was not intended. Happy to revert.

Reply:
No I think consistency is good. So long as it's working I think this is okay :)

Reply:
We are using different wording for `AccountCredential` and `SetCredentialRequest` though, like:
`AccountCredential::RemoveWebauthn` vs `SetCredentialRequest::WebauthnRemove`

So perhaps we should also rename the other Enum in AccountCredential to follow SetCredentialRequest?

Reply:
Yeah, don't see a problem with this :)

Reply:
So do you want we to do more of these renames?
AccountCredential::RemoveWebauthn vs SetCredentialRequest::WebauthnRemove

How should it be? [Action][CredType] or [CredType][Action]
Have a separate ticket?